README.md

```diff
@@ -52,6 +52,20 @@ All of our chosen linting rules, tests, as well as package builds can be execute
 #### Strict mode
 TypeScript features optional static typing via [strict mode](https://www.typescriptlang.org/tsconfig/#strict), which has been enabled for our example code.
 
+### Security scans
```

Comment:
So instead of a new section, in my PR I added these to the existing table under a new column. I'm not sure what the best  way to go about aligning our approaches for updating the readme with security details 🤔 e.g. here is mine: 

https://github.com/British-Oceanographic-Data-Centre/amrit-repos/blob/feature/python-security-scans/README.md